content/en/real_user_monitoring/session_replay/privacy_options.md

```diff
@@ -59,7 +59,7 @@ Masks most form fields such as inputs, text areas, and checkbox values while rec
 
 ### Mask mode
 
-Setting `defaultPrivacyLevel` to `mask` mode will mask all HTML text, user input, images, and links. Text on your application is replaced with `X`, rendering the page into a wireframe.
+Setting `defaultPrivacyLevel` to `mask` mode will mask all HTML text, user input, images, links and [`data-*` attributes][1]. Text on your application is replaced with `X`, rendering the page into a wireframe.
```

Comment:
```suggestion
Setting `defaultPrivacyLevel` to `mask` mode masks all HTML text, user input, images, links and [`data-*` attributes][1]. Text on your application is replaced with `X`, rendering the page into a wireframe.
```

Reply:
Feel free to commit this change and merge yourself, I didn't edit pre-existing copy.